internal/reconcile/daemon_sets.go

```diff
@@ -33,8 +33,8 @@ func DaemonSet(er record.EventRecorder, k8Client client.Client, desired *apps.Da
 		}
 		same := false
 
-		if same, updateReason = daemonsets.AreSame(current, desired); same && util.HasSameOwner(current.OwnerReferences, desired.OwnerReferences) {
-			log.V(3).Info("DaemonSet are the same skipping update")
+		if same, updateReason = pod.AreSame(&current.Spec.Template.Spec, &desired.Spec.Template.Spec, current.Name); same && util.HasSameOwner(current.OwnerReferences, desired.OwnerReferences) {
```

Comment:
similar ask here as deployments.  We are now possibly missing items we wish to keep managed that are specific to a daemonset

Reply:
See comment above